### PR TITLE
Resolved the user not being able to retrieve their customer information

### PR DESCRIPTION
**Pull request by oiricaud**
## Pull request summary
This pull request has minor changes that pertain on the ability to allow the user to retrieve their profile information.

### CODE REVIEW

File: public/resources/components/services/BlueAPIService.js [hunk 1]
```diff
@@ -72,8 +72,9 @@ app.service('BlueAPIService',['$http', 'CONFIG', '$base64', function($http, CONF
 				invokeService(restUrl, requestType, null, successCallback, errorCallback);
 			},
 			loginUser : function(parameters, successCallback, errorCallback) {
-				//var restUrl = CONFIG["Auth-Server"].protocol + '://' + CONFIG["Auth-Server"].host + '/oauth/token'
-				var restUrl = 'oauth/token'
+				// where port 8080=auth microservice
+				var restUrl = 'http://localhost:8080/oauth/token'
```
Comment (phemankita) [inline public/resources/components/services/BlueAPIService.js:76]: `var restUrl = 'oauth/token'` shd work. Do we need the entire url here ?

File: public/resources/components/services/BlueAPIService.js [hunk 2]
```diff
@@ -88,7 +89,8 @@ app.service('BlueAPIService',['$http', 'CONFIG', '$base64', function($http, CONF
 				invokeService(restUrl, requestType, parameters, successCallback, errorCallback, access_token);
 			},
 			getCustomerProfile : function(access_token, successCallback, errorCallback) {
-				var restUrl = 'customer/';
+				// where port 8080=auth microservice
+				var restUrl = 'http://localhost:8080/customer'
```
Comment (phemankita) [inline public/resources/components/services/BlueAPIService.js:93]: `var restUrl = 'oauth/token'` shd work. Do we need the entire url here ?

File: public/resources/components/services/BlueAPIService.js (hunk 1 above)
Comment (oiricaud) [reply public/resources/components/services/BlueAPIService.js:76]: When testing locally yes

Comment (oiricaud) [reply public/resources/components/services/BlueAPIService.js:76]: `localhost:8080` was the `auth-ms-spring` I wasn't sure how to test it without hardcoding the `restUrl`

File: public/resources/components/services/BlueAPIService.js (hunk 2 above)
Comment (oiricaud) [reply public/resources/components/services/BlueAPIService.js:93]: I am confused... should this be calling the `http://auth-ms-spring:8080/customer` or `http://auth-ms-spring:8080/oauth/token`

**Comment by oiricaud** [comment]
@phemankita I just realized that the `BlueAPIService.js` class is calling the `web` microservice for the `oauth/token` rest end point. Trying to figure out how to pass in the environment variables to call the auth microservice instead. 

```
var restUrl = 'http://' + auth_host + ':' + auth_port + '/oauth/token'
```

**Comment by oiricaud** [comment]
@phemankita I have resolved the issues you brought up, if you could review and merge this :)

**Review by phemankita — APPROVED** [review]
LGTM

**Pull request merged**